source/docs/articles/sites/domains/adding-a-ssl-certificate-for-secure-https-communication.md

```diff
@@ -1,27 +1,27 @@
 ---
-title: Enable SSL for Secure HTTPS Communication
-description: Learn how to implement SSL secure HTTPS communication and utilize a static IP address.
+title: Enable Secure HTTPS Communication
+description: Learn how to implement secure HTTPS communication and utilize a static IP address.
 category:
   - developing
   - launch
-keywords: secure https, https, ssl, security, encryption, enable ssl, enable ssl certificate, add ssl cert, add ssl, add https, add encryption, how to add an ssl, dns, csr, generate csr, generate key, update dns for ssl, ssl certificate types, ssl cert types, ssl types, ssl provider, intermediary ssl, intermediary ssl cert, intermediary ssl certificate
+keywords: secure https, https, security, encryption, add ssl cert, add https, add encryption, dns, csr, generate csr, generate key, update dns for ssl, ssl certificate types, ssl cert types,  ssl provider, intermediary ssl, intermediary ssl cert, intermediary ssl certificate
 ---
-SSL is a standard for establishing an encrypted link between your Pantheon site and a client (e.g. web browser). You should enable SSL on a custom domain, e.g., www.example.com, if you are transmitting any sensitive data. Loading a valid OpenSSL certificate into a Pantheon environment provisions an SSL load balancer with a dedicated IP address, allowing secure communication over HTTPS. All traffic within the Pantheon infrastructure, from Varnish to application containers, is encrypted.
+HTTPS is a standard for establishing an encrypted link between your Pantheon site and a client (e.g. web browser). You should enable HTTPS on a custom domain, e.g., www.example.com, if you are transmitting any sensitive data. Loading a valid OpenSSL certificate into a Pantheon environment provisions an SSL load balancer with a dedicated IP address, allowing secure communication over HTTPS. All traffic within the Pantheon infrastructure, from Varnish to application containers, is encrypted.
 
-Adding SSL to your site is a completely self-serve option; Pantheon does not provide private keys or certificate signing requests (CSRs), or any SSH login for you to generate these. The key and certificates are cryptographically sensitive elements that you should not send through email, as cleartext is very insecure.
+Adding HTTPS to your site is a completely self-serve option; Pantheon does not provide private keys or certificate signing requests (CSRs), or any SSH login for you to generate these. The key and certificates are cryptographically sensitive elements that you should not send through email, as cleartext is very insecure.
 
 <div class="alert alert-danger" role="alert">
 <h4>Warning</h4>
-Enable SSL before updating DNS. SSL for custom domains is available for Professional plans and above.</div>
+Enable HTTPS before updating DNS. HTTPS for custom domains is available for Professional plans and above.</div>
 
-## Steps to Enable SSL
+## Steps to Enable HTTPS
 
 1. Generate RSA Key and CSR
 2. Get SSL certificate
```

Comment:
Just a certificate. It's not technically an SSL, TLS, or HTTPS certificate. It's actually an X.509 one (but that's probably not worth mentioning).